ArduCopter/GCS_Mavlink.cpp

```diff
@@ -995,30 +995,29 @@ MAV_RESULT GCS_MAVLINK_Copter::handle_command_long_packet(const mavlink_command_
 
 MAV_RESULT GCS_MAVLINK_Copter::handle_command_pause_continue(const mavlink_command_int_t &packet)
 {
-#if MODE_AUTO_ENABLED
-    if (copter.flightmode->mode_number() != Mode::Number::AUTO) {
-        // only supported in AUTO mode
+#if defined(MODE_GUIDED_ENABLED) || defined(MODE_AUTO_ENABLED)
+    // requested pause
+    if ((uint8_t) packet.param1 == 0) {
+        if (copter.flightmode->pause()) {
+            send_text(MAV_SEVERITY_INFO, "Paused");
```

Comment:
Let's remove this "Paused" send text.  In general I'd like to reduce the text messages we send when things are working properly.  The user has likely just pushed the button and they should see the vehicle pause.

In the case where it fails (just below) we should keep send-text.